packages/ui/src/components/Banner/banner.tsx

```diff
@@ -47,22 +85,25 @@ export const Banner: React.FC<BannerProps> = ({
     imgUrl,
     invertText,
     id,
+    size = 'medium',
 }) => {
     const [{ width }, setRef] = useContainerSize<HTMLDivElement>()
 
-    const showImage = imgUrl && width >= MD_BARRIER
+    const showImage = getShowImage(width, size, imgUrl)
 
     const bannerClasses = classNames({
         [`${CLASS_PREFIX}`]: true,
         [`${CLASS_PREFIX}-no-image`]: !showImage,
+        [`${CLASS_PREFIX}-compact`]: size === 'small',
```

Comment:
Why different terminology is used (compact and small)? Why not just `[${CLASS_PREFIX}-${size}]: size`?